app/helpers/contextCreators/generateAddPriceTable.test.js

```diff
@@ -59,7 +59,7 @@ describe('generateAddPriceTable', () => {
     };
 
     const generatedAddPriceTable = generateAddPriceTable({
-      addPriceTable, price: '1892.23', itemUnitDescription: 'per patient', timeUnitDescription: 'per year',
+      addPriceTable, price: 1892.23, itemUnitDescription: 'per patient', timeUnitDescription: 'per year',
```

Comment:
Does this test represent data from BE, hence why it is a number & not a string?

Reply:
yeah it comes from the BE as a number. So updated the test data to reflect that